src/styles/app.css

```diff
@@ -53,11 +71,17 @@ h4{
   font-family: 'Nunito', sans-serif;
 }
 
+label{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+  color: #185253;
+ }
 input{
```

Comment:
Consider adding an empty line between 78 and 79.  

Also be consistent with the spacing here:

Line 79 - input{
Line 87 - .nav-btn {

I had to look pretty hard to find stuff, good work